releasenotes/notes/add-maas-checks-for-filbeat-es-rsyslog-0289a1d9a1c198d0.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    This adds MaaS checks for elasticsearch, beaver, and 
```

Comment:
Yes, tracking release notes within the repo was introduced with Mitaka. A few have already slipped through into previous branches but they will not be used in any way.